blocks/library/cover-image/editor.scss

```diff
@@ -22,4 +22,16 @@
 	.components-placeholder h2 {
 		color: inherit;
 	}
+
+	&.contentalignleft {
```

Comment:
Let's use classes like `has-left-content` here as it's easier to control and be consistent (modifier classes like `is-` and `has-`) when we apply to `.block.has-left-content`.

Reply:
I kind of assumed Jorge was keeping with the WordPress convention of `alignleft` _et al._. Should we push for change in that area too?

Reply:
I have left the alignleft / right / center untouched for now, but any new class we should take care of it. (Also applies to columns as something we need to address.)